Specs/Core/JulianDateSpec.js

```diff
@@ -166,75 +166,75 @@ defineSuite([
 
     it('Construct from ISO8601 local calendar date, basic format', function() {
         var expectedDate = JulianDate.fromDate(new Date(2009, 7, 1));
-        var computedDate = JulianDate.fromIso8601('20090801');
+        var computedDate = JulianDate.fromIso8601('20090801T00');
```

Comment:
Since these tests were originally meant to test the functionality you're changing, I would recommend removing the word local from each test and using `new Date(Date.UTC(` instead of changing the values.  Then you can remove your new tests because they will be covered by these old ones.